spec/tapioca/compilers/dsl/active_record_associations_spec.rb

```diff
@@ -396,4 +396,138 @@ def commenters_attributes=(attributes); end
       assert_equal(expected, rbi_for(:Post))
     end
   end
+
+  describe("#decorate_active_storage") do
+    before(:each) do
+      require "active_record"
+      require "active_storage/attached"
+      require "active_storage/reflection"
+      ActiveRecord::Base.include(ActiveStorage::Attached::Model)
+      ActiveRecord::Base.include(ActiveStorage::Reflection::ActiveRecordExtensions)
+      ActiveRecord::Reflection.singleton_class.prepend(ActiveStorage::Reflection::ReflectionExtension)
+      ActiveRecord::Base.establish_connection(
+        adapter: "sqlite3",
+        database: ":memory:"
+      )
+
+      T.bind(self, Tapioca::Compilers::Dsl::ActiveRecordAssociationsSpec)
+      add_ruby_file("application.rb", <<~RUBY)
+        ENV["DATABASE_URL"] = "sqlite3::memory:"
+
+        require "active_storage/engine"
+
+        class Dummy < Rails::Application
```

Comment:
creating a dummy app to run all Rails initializer to load all ActiveStorage classes.